edge-modules/MetricsCollector/IMetricsSync.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+namespace MetricsCollector
+{
+    using System.Threading.Tasks;
+
+    interface IMetricsSync
```

Comment:
should it be IMetricSubscriber?

Reply:
It's not really subscribing to metrics updates, but scraping and syncing them itself.